src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="card">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0
+    ">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="card__content">
+    <div class="card-block" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="img-imac"
+        class="card-block__img"
+      >
+
+      <p class="card-block__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+      <p class="card-block__code">Код товара: 195434</p>
+
+      <div class="card-block__rate">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card-block__reviews">
+          Отзывов: 5
+        </div>
+      </div>
+      <div class="card-block__container-price">
+        <div class="card-block__price">Цена:</div>
+        <div class="card-block__price-value">69 999 грн</div>
+      </div>
+      <a
+        href="#"
+        data-qa="hover"
+        class="card-block__button-link">
+        <div class="card-block__button">Купить</div>
+      </a>
```

Comment:
Don't need add div element inside a

```suggestion
      <a
        href="#"
        data-qa="hover"
        class="card-block__button-link"
      >
        Купить
      </a>
```